app/src/main/java/com/garethevans/church/opensongtablet/PopUpChordsFragment.java

```diff
@@ -84,7 +85,14 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         View V = inflater.inflate(R.layout.popup_page_chords, container, false);
 
         TextView title = V.findViewById(R.id.dialogtitle);
-        title.setText(Objects.requireNonNull(getActivity()).getResources().getString(R.string.chords));
+
+        // Title changed to reflect Native or Capo chord display
+        if (StaticVariables.showCapoInChordsFragment) {
+            title.setText(Objects.requireNonNull(getActivity()).getResources().getString(R.string.showcapo));
+        } else {
+            title.setText(Objects.requireNonNull(getActivity()).getResources().getString(R.string.showchords));
+        }
+
```

Comment:
I really like this suggestion, however, it doesn't seem to do anything yet?  Need to figure out where/how StaticVariables.showCapoInChordsFragment is set

Reply:
It is in the long press for 'show capo chords' button.

The menu may need an option to display capo chords.  The separate 'chords' button could be retired as not necessary. That is menu stuff which I did not touch.

Reply:
Got it.  I think leave it as it is for now.